var/spack/repos/builtin/packages/gunrock/package.py

```diff
@@ -12,7 +12,11 @@ class Gunrock(CMakePackage, CudaPackage):
     homepage = "https://gunrock.github.io/docs/"
     git      = "https://github.com/gunrock/gunrock.git"
 
+    # tagged versions are broken. See
+    # https://github.com/gunrock/gunrock/issues/777
+    # Hence, prefer a specific commit.
     version('master',   submodules=True)
+    version('2020-06-15',   submodules=True, commit='81f58d628463561969dafe65868e72251562e806', preferred=True)
```

Comment:
I wonder if having a version like this one alongside the others below respecting semver would be problematic :thinking: @adamjstewart Do you have any experience of another package like this?

Reply:
It won't be problematic until v1.2 is released, then things won't sort correctly.

Reading through https://github.com/gunrock/gunrock/issues/777, it seems like the author is willing to help us write a patch to get the previous releases working. I would wait on that, especially since users can always install `master` until this is fixed.

Reply:
I was planning to remove it when the patch became available. I can just remove it now. Master version is good enough I guess.